common/common-api/src/main/java/org/ow2/proactive/utils/FileUtils.java

```diff
@@ -47,6 +49,8 @@
 @PublicAPI
 public class FileUtils {
 
+    private final static Logger logger = Logger.getLogger(FileUtils.class);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Reorder the modifiers to comply with the Java Language Specification. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AModifiersOrderCheck)